tests/resources/serializers/test_marcxml_serializer.py

```diff
@@ -100,72 +103,67 @@ def test_marcxml_serializer_full_record(running_app, updated_full_record):
-  </datafield>
-  <datafield tag="520" ind1=" " ind2="2">
-    <subfield code="a">11 pages</subfield>
-  </datafield>
-  <datafield tag="653" ind1=" " ind2=" ">
-    <subfield code="a">custom</subfield>
-  </datafield>
-  <datafield tag="100" ind1="a" ind2=" ">
-    <subfield code="a">Nielsen, Lars Holm</subfield>
-  </datafield>
-  <datafield tag="260" ind1="b" ind2=" ">
-    <subfield code="a">InvenioRDM</subfield>
-  </datafield>
-  <datafield tag="901" ind1=" " ind2=" ">
-    <subfield code="u">info:eu-repo/semantic/other</subfield>
-  </datafield>
-  <datafield tag="245" ind1="a" ind2=" ">
-    <subfield code="a">InvenioRDM</subfield>
-  </datafield>
-  <datafield tag="520" ind1=" " ind2=" ">
-    <subfield code="a">A description 
-with HTML tags</subfield>
-    <subfield code="a">Bla bla bla</subfield>
-  </datafield>
-  <datafield tag="856" ind1=" " ind2="2">
-    <subfield code="a">doi:10.1234/foo.bar</subfield>
-  </datafield>
-  <datafield tag="260" ind1="c" ind2=" ">
-    <subfield code="c">2018/2020-09</subfield>
-    <subfield code="c">info:eu-repo/date/embargoEnd/2131-01-01</subfield>
-  </datafield>
-  <datafield tag="024" ind1=" " ind2="3">
-    <subfield code="a">v1.0</subfield>
-  </datafield>
-  <datafield tag="024" ind1=" " ind2=" ">
-    <subfield code="a">1924MNRAS..84..308E</subfield>
-    <subfield code="a">10.5281/inveniordm.1234</subfield>
-  </datafield>
-  <datafield tag="540" ind1=" " ind2=" ">
-    <subfield code="a">info:eu-repo/semantics/embargoedAccess</subfield>
-    <subfield code="a">A custom license</subfield>
-    <subfield code="a">https://customlicense.org/licenses/by/4.0/</subfield>
-    <subfield code="a">Creative Commons Attribution 4.0 International</subfield>
-    <subfield code="a">https://creativecommons.org/licenses/by/4.0/legalcode</subfield>
-  </datafield>
-  <datafield tag="856" ind1=" " ind2="1">
-    <subfield code="a">award_identifiers_scheme=null; award_identifiers_identifier=null; award_title=null; award_number=null; funder_id=00k4n6c32; funder_name=null; </subfield>
-  </datafield>
-</record>
-    """
+    expected_data = "\n".join(
+        [
+            '<record xmlns="http://www.loc.gov/MARC21/slim">',
+            '  <datafield tag="700" ind1="a" ind2=" ">',
+            '    <subfield code="u">Nielsen, Lars Holm</subfield>',
+            "  </datafield>",
+            '  <datafield tag="540" ind1=" " ind2=" ">',
+            '    <subfield code="a">info:eu-repo/semantics/embargoedAccess</subfield>',
+            '    <subfield code="a">A custom license</subfield>',
+            '    <subfield code="a">https://customlicense.org/licenses/by/4.0/</subfield>',
+            '    <subfield code="a">Creative Commons Attribution 4.0 International</subfield>',
+            '    <subfield code="a">https://creativecommons.org/licenses/by/4.0/legalcode</subfield>',
+            "  </datafield>",
+            '  <datafield tag="024" ind1=" " ind2="3">',
```

Comment:
according to the issue the pids should be splited. However, I do not see the doi of the record (see [fixture](https://github.com/inveniosoftware/invenio-rdm-records/blob/5bdf56cf30115207648f61661e5e0378cb0a4c6f/tests/conftest.py#L371)) in the serialization. I **think** (not 100% sure) that the result should be something like:

```
<datafield tag="024" ind1=" " ind2=" ">
        <subfield code="a">10.5281/inveniordm.1234</subfield>
        <subfield code="2">doi</subfield>
</datafield>
```

Reply:
I think I'm missing something. Are DOI and OAI being tested in `024` fields, I think it should be tested. I can only find the `v1` identifiers in the tests. We can chat or have a quick call to clarify this if needed.